programs/voter-stake-registry/src/state/voting_mint_config.rs

```diff
@@ -18,11 +18,15 @@ pub struct VotingMintConfig {
     /// The authority that is allowed to push grants into voters
     pub grant_authority: Pubkey,
 
-    /// Vote weight factor for unlocked deposits, in 1/SCALED_FACTOR_BASE units.
-    pub unlocked_scaled_factor: u64,
+    /// Vote weight factor for all funds in the account, no matter if locked or not.
+    /// In 1/SCALED_FACTOR_BASE units.
+    pub baseline_vote_weight_scaled_factor: u64,
 
-    /// Maximum vote weight factor for lockups, in 1/SCALED_FACTOR_BASE units.
-    pub lockup_scaled_factor: u64,
+    /// Maximum extra vote weight factor for lockups. This is the extra votes gained
+    /// for lockups lasting lockup_saturation_secs or longer. For shorter lockups only
+    /// a fraction of it is used.
```

Comment:
nitpick: please elaborate on  - For shorter lockups only a fraction of it is used -